insights/parsers/crypto_policies.py

```diff
@@ -0,0 +1,89 @@
+"""
+crypto-policies - files in ``/etc/crypto-policies/back-ends/``
+==============================================================
+
+This is a collection of parsers that all deal with the generated configuration
+files under the ``/etc/crypto-policies/back-ends/`` folder.  Parsers included
+in this module are:
+
+CryptoPoliciesConfig - file ``/etc/crypto-policies/config``
+-----------------------------------------------------------
+
+CryptoPoliciesStateCurrent - file ``/etc/crypto-policies/state/current``
+------------------------------------------------------------------------
+
+CryptoPoliciesOpensshserver - file ``/etc/crypto-policies/back-ends/opensshserver.config``
+------------------------------------------------------------------------------------------
+"""
+
+from insights import Parser, parser, SysconfigOptions
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+
+@parser(Specs.crypto_policies_config)
+class CryptoPoliciesConfig(Parser):
+    """
+    This parser reads the ``/etc/crypto-policies/config`` file.  It uses the
+    ``SysconfigOptions`` parser class to convert the file into
+    a dictionary of options. It also provides the ``options`` property as a helper
+    to retrieve the ``CRYPTO_POLICY`` variable.
```

Comment:
I believe your docstring is incorrect, it is based on `Parser` not `SysconfigOptions`.  It also refers to an `options` attribute which is not present.